src/package/vsix/CMakeLists.txt

```diff
@@ -14,7 +14,7 @@ file(TO_NATIVE_PATH "${cppwinrt_nupkg_dir}/Microsoft.Windows.CppWinRT.${XLANG_BU
 
 file(DOWNLOAD https://dist.nuget.org/win-x86-commandline/latest/nuget.exe ${nuget_exe})
 
-set(build_vsix msbuild ${CMAKE_CURRENT_SOURCE_DIR}/vsix.csproj /nologo /m /p:Configuration=${CMAKE_BUILD_TYPE},Platform=$ENV{VSCMD_ARG_TGT_ARCH})
+set(build_vsix msbuild ${CMAKE_CURRENT_SOURCE_DIR}/vsix.csproj /nologo /m /p:Configuration=${MSBUILD_CONFIGURATION},Platform=$ENV{VSCMD_ARG_TGT_ARCH})
```

Comment:
Blarg. Also need to pass in CmakeBuildType here as well since the pipeline uses build configuration for paths